lib/std/fmt/format_float.zig

```diff
@@ -63,22 +63,32 @@ pub fn formatFloat(buf: []u8, v_: anytype, options: FormatOptions) FormatError![
     comptime std.debug.assert(@typeInfo(T) == .Float);
     const I = @Type(.{ .Int = .{ .signedness = .unsigned, .bits = @bitSizeOf(T) } });
 
+    const DT = if (@bitSizeOf(T) <= 64) u64 else u128;
+    const tables = switch (DT) {
+        u64 => if (@import("builtin").mode == .ReleaseSmall) &Backend64_TablesSmall else &Backend64_TablesFull,
+        u128 => &Backend128_Tables,
+        else => unreachable,
+    };
+
     const has_explicit_leading_bit = std.math.floatMantissaBits(T) - std.math.floatFractionalBits(T) != 0;
-    const d = binaryToDecimal(@as(I, @bitCast(v)), std.math.floatMantissaBits(T), std.math.floatExponentBits(T), has_explicit_leading_bit);
+    const d = binaryToDecimal(DT, @as(I, @bitCast(v)), std.math.floatMantissaBits(T), std.math.floatExponentBits(T), has_explicit_leading_bit, tables);
```

Comment:
Should probably place the comptime tables as the second argument here instead of last.